### ISSUE TITLE
[BUG] - Can't See Bids in My Flights

### ISSUE THREAD

**Issue by walkerair** [invernyx/smartcars-3-bugs#73]
**Describe the bug**
My Flights is blank.

![image](https://user-images.githubusercontent.com/127605985/229603716-75a8ee83-a25d-42a6-8d25-703cfdf8f413.png)

**To Reproduce**
1. Log-in
2. Go to My Flights

**Expected behavior**
I should see my active bids.

I have deleted/recreated flights multiple times.

I put a trace on bookings.php and I see the database pull coming back with data but it doesn't seem to appear in the app. Error.log and Combined.log don't give me any indications.

[{"bidID":251862,"code":"WAT","number":"1682","type":"P","departureAirport":"KATL","arrivalAirport":"KEVV","route":"","aircraft":"1","flightLevel":"","departureTime":"7:49","arrivalTime":"9:07","flightTime":"0.90","distance":"305","daysOfWeek":"1234560","notes":""}]

**Screenshots**
N/A

**System Information (please complete the following information):**
 - OS: Windows 11
 - Community ID: [e.g. 2/any]
 - Version Latest
 - Plugins used: [e.g. chat, flight-breakdown, flight-center, flight-tracking, map-plugin, pilot-center]

**Additional context**
It was working fine the other day -- I actually logged a flight with it.

**Comment by walkerair** [comment]
I did a flight on April 1st, no issues.

Now, I'm getting this when I do an API test.

{
    "message": "The handler provided could not be found"
}

No files have changed. No locations on my server have changed.

Combined log now shows this:

info: Received notification {"handler":"event://notification","notification":{"id":"0fb6763c-cdb7-4bb6-8ef0-bb32efce158a","message":"","source":"flight-center","timestamp":1680552602540,"toast":{"message":"Failed to get bid flights","type":"danger"},"type":"info"},"source":"realTimeAPI"}
info: Received notification {"handler":"event://notification","notification":{"id":"d72321c3-cf6e-4707-9737-c2013467e008","message":"","source":"flight-center","timestamp":1680552616817,"toast":{"message":"Failed to get bid flights","type":"danger"},"type":"info"},"source":"realTimeAPI"}
info: Received notification {"handler":"event://notification","notification":{"id":"fc1399c3-160e-42d3-a62d-3e440f14cdfd","message":"","source":"flight-center","timestamp":1680552619044,"toast":{"message":"Failed to get bid flights","type":"danger"},"type":"info"},"source":"realTimeAPI"}
info: Received notification {"handler":"event://notification","notification":{"id":"2d0e82db-647b-4e07-bf43-308339348ef4","message":"","source":"flight-center","timestamp":1680553221773,"toast":{"message":"Failed to get bid flights","type":"danger"},"type":"info"},"source":"realTimeAPI"}

And the app console shows this:

![image](https://user-images.githubusercontent.com/127605985/229619163-4e240c6a-11eb-4f0b-bef4-2f7fc868e399.png)

Not sure where to start.

I can log in just fine to the app, and I can also wipe my session in the database and sign back in (get a new session.)

**Comment by walkerair** [comment]
Been talking to Fabio, but here are screens in case you need them. Token is removed (because obviously I don't want people seeing it.) The API is working fine -- I can query everything.

![image](https://user-images.githubusercontent.com/127605985/229652642-c3f9a035-d0d2-4634-8336-c0c516513260.png)

![image](https://user-images.githubusercontent.com/127605985/229652726-da73832b-d4f8-47f1-ab08-4c54b83275f3.png)

I can log into the app just fine tho.

**Comment by walkerair** [comment]
Disregard. I know what the issue is and it's not your side. Apologies for this.

**Comment by jordanhavard** [comment]
> Disregard. I know what the issue is and it's not your side. Apologies for this.

Are you able to share what the issue was (helps me and other prevent the issue if they come across this)

**Comment by walkerair** [comment]
> 

I basically run my aircraft tables a lot different than phpVMS 5/7 -- I forgot about that (brain fart) and realized there are times when the aircraft ID = 1 in the schedule row. So, I just had to rewrite the bookings.php database call to present a different table for the aircraft really used on the flight so that it could match up. The data was pulled but it could not understand what aircraft ID = 1 was (since that doesn't exist.) And therefore it wouldn't show that particular flight. I only recognized it because I was looking at the JSON above and noticed the =1. :)

I'm all good now and back in business.
